.github/workflows/deploy.yml

```
@@ -71,7 +72,11 @@ jobs:
           script: |
             DOCKER_IMAGE="${{ secrets.DOCKER_USERNAME }}/${{ secrets.DOCKER_REPOSITORY_NAME }}"
 
-            echo "${{ secrets.ENV }}" > prod.env
+            cat > prod.env << EOF
+            ${{ secrets.ENV }}
+            EOF
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)